scripts/nbconverted/error_geneExp_reconstruction.py

```diff
@@ -0,0 +1,103 @@
+
+# coding: utf-8
+
+# In[1]:
+
+
+get_ipython().run_line_magic('matplotlib', 'inline')
+
+
+# In[2]:
+
+
+#-------------------------------------------------------------------------------------------------------------------------------
+# By Alexandra Lee (July 2018) 
+#
+# Calculate error between reconstructed vs observed gene expression
+#-------------------------------------------------------------------------------------------------------------------------------
+import os
+import pandas as pd
+import numpy as np
+import seaborn as sns
+np.random.seed(123)
+
+
+# In[3]:
+
+
+# load arguments
+estimated_file = os.path.join(os.path.dirname(os.getcwd()), "output", "estimated_test_control_latent.txt")
+obsv_file = os.path.join(os.path.dirname(os.getcwd()), "data", "test_treat.txt")
+
+# If using latent space estimation
+latent = True
+
+# output
+fig_file = os.path.join(os.path.dirname(os.getcwd()), "viz", "estimated_sample0_latent.png")
+
+
+# In[4]:
+
+
+# read in data
+estimated_data = pd.read_table(estimated_file, header = 0, sep = '\t', index_col = 0)
+obsv_data = pd.read_table(obsv_file, header = 0, sep = '\t', index_col = 0)
+
+if latent:
+    estimated_data.columns = obsv_data.columns
+
+#obsv_data.head(5)
+#estimated_data.head(5)
+
+
+# In[5]:
+
+
+# Average gene expression across samples
+estimated_data_mean = estimated_data.mean(axis=0)
+obsv_data_mean = obsv_data.mean(axis=0)
+
+#estimated_data_mean_df = pd.DataFrame(estimated_data_mean, index = estimated_data_mean.index, columns=['estimated'])
+#obsv_data_mean_df = pd.DataFrame(obsv_data_mean, index = obsv_data_mean.index, columns = ['observed'])
+
+# Try using single sample
+estimated_data_mean_df = pd.DataFrame(estimated_data.iloc[0].transpose())
+obsv_data_mean_df = pd.DataFrame(obsv_data.iloc[0].transpose())
+estimated_data_mean_df.columns = ['estimated']
+obsv_data_mean_df.columns = ['observed']
+
+estimated_data_mean_df.head(5)
+#obsv_data_mean_df.head(5)
+
```

Comment:
also here

Reply:
This is for when I was spot checking what the data looks like